scripts/speak.py

```diff
@@ -50,11 +67,15 @@ def macos_tts_speech(text):
     os.system(f'say "{text}"')
 
 def say_text(text, voice_index=0):
-
+    
```

Comment:
You introduced trailing space. Remove it. (It sometimes creeps in during merges)